src/JuMP.jl

```diff
@@ -384,7 +384,7 @@ end
 
 Returns number of variables in `model`.
 """
-num_variables(model::Model) = MOI.get(model, MOI.NumberOfVariables())
+num_variables(model::Model) = MOI.get(model, MOI.NumberOfVariables())::Int64
```

Comment:
What if a solver returns an Int32?

Reply:
We should use a conversion here, that is also consistent with `num_constraint` in https://github.com/JuliaOpt/JuMP.jl/pull/1850/